src/Extensions/Extensions.csproj

```diff
@@ -6,7 +6,7 @@
     <RootNamespace>Microsoft.Azure.Functions.Extensions</RootNamespace>
     <PackageId>Microsoft.Azure.Functions.Extensions</PackageId>
     <Description>Provides extensions to work with Azure Functions features.</Description>
-    <Version>1.2.0$(VersionSuffix)</Version>
+    <Version>1.2.0</Version>
```

Comment:
We may want to switch to `VersionPrefix` to retain the ability to pass in `VersionSuffix` from cmd